charts/osm-arc/values.yaml

```diff
@@ -28,7 +28,7 @@ osm:
     deployPrometheus: false
     deployJaeger: false
     webhookConfigNamePrefix: arc-osm-webhook
-    sidecarImage: mcr.microsoft.com/oss/envoyproxy/envoy:v1.28.7
+    sidecarImage: mcr.microsoft.com/oss/envoyproxy/envoy-distroless:v1.32.2-hotfix.20241216
```

Comment:
Not sure we should move to distroless here since previous versions have had a shell. WDYT?